cms/static/cms/js/require-config.js

```diff
@@ -257,6 +258,9 @@
             "afontgarde": {
                 exports: "AFontGarde"
             },
+            "paging-collection": {
```

Comment:
I'm surprised that this is necessary because the collection class uses RequireJS. Can you try removing it to see.